frontend/pages/SoftwarePage/SoftwareTitleDetailsPage/SoftwareInstallerCard/SoftwareInstallerCard.tsx

```diff
@@ -136,10 +138,10 @@ const SoftwareActionButtons = ({
     );
     options = options.map((option) => {
       // edit is disabled in gitOpsMode for VPP only
-      // delete is disabled in gitOpsMode for all installers (FMA, VPP, & custom packages)
+      // delete is disabled in gitOpsMode for FMA and VPP onlu
```

Comment:
```suggestion
      // delete is disabled in gitOpsMode for software types that can't be added in GitOps mode (FMA, VPP)
```

Was wondering why we were singling out custom packages for this but realized the above comment is why, so this feels like it'll be helpful for the next person who looks at the code so _they_ don't think this is a UI consistency bug.